replayer/neard/neard.go

```diff
@@ -7,20 +7,52 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/aurora-is-near/evm-bully/util/git"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/frankbraun/codechain/util/file"
 )
 
-// NEARDaemon wraps a running neard.
+// NEARDaemon wraps a neard.
 type NEARDaemon struct {
 	Head       string
+	binaryPath string
+	localDir   string
 	nearDaemon *exec.Cmd
 }
 
-// Build builds neard in CWD.
-func Build(release bool) error {
+func getDefaultLocalDir() (string, error) {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(home, ".near", "local"), nil
+}
+
+// LoadFromBinary loads NEARDaemon from existing binary.
+func LoadFromBinary(binaryPath string, head string) (*NEARDaemon, error) {
+	log.Info("load neard binary")
+
+	if _, err := os.Stat(binaryPath); err != nil {
+		return nil, fmt.Errorf("can't access neard binary on path %v: %v", binaryPath, err)
```

Comment:
By conventions, errors should start with the package they originate from. So "neard: " in this case.

See https://golang.org/doc/effective_go#errors